src/styles/product-card.scss

```diff
@@ -0,0 +1,87 @@
+.card {
+  width: 198px;
+  height: 374px;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  color: $main;
+  padding-block: 32px 0;
+
+  &__image,
+  &__descreption,
+  &__code,
+  &__reviews,
+  &__price,
+  &__button {
+    width: 166px;
+    display: flex;
+    align-items: center;
+    margin: 0 auto;
+  }
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    margin-bottom: 40px;
+    padding-inline: 3px;
+  }
+
+  &__descreption {
+    font-size: 1.2rem;
+    font-weight: 500;
+    line-height: 18px;
+    margin-bottom: 4px;
+  }
+
+  &__code {
+    justify-content: start;
```

Comment:
This code is redundant as your string will display from start by default

Reply:
Thanks, solved.